.github/workflows/main.yml

```
@@ -100,7 +100,7 @@ jobs:
         prerelease: ${{ contains(steps.get_version.outputs.version, '-') }}
     - name: Setup .NET Core
       if: steps.check_tag.outputs.statusCode == '404'
-      uses: actions/setup-dotnet@v1
+      uses: actions/setup-dotnet@v2
```

```suggestion
      uses: actions/setup-dotnet@v1
```

`setup-dotnet` is v1.